pwa-devdocs/src/tutorials/create-taglist-component/index.md

```diff
@@ -0,0 +1,454 @@
+Open the `tagList.css` file and add the following content:
+
+```css
+.root {
+    display: flex;
+    flex-wrap: wrap;
+}
+```
+
+This entry defines the root container of a TagList component as a [flexbox][] that wraps.
+This prevents the Tags from overflowing off the side of the page.
+
+#### `tagList.js`
+
+Open the `tagList.js` file and make the following changes to the component.
+
+**Change 1:** Import the `mergeClasses()` function and the `tagList.css` file:
+
+```diff
+ import { arrayOf, shape, string } from 'prop-types';
+ 
++import { mergeClasses } from '../../classify';
++import defaultClasses from './tagList.css';
+ 
+ import Tag from './tag';
+```
+
+**Change 2:** Use the `mergeClasses()` function to merge custom classes passed into the component:
+
+```diff
+ const { categories } = props;
+ 
++const classes = mergeClasses(defaultClasses, props.classes);
+```
+
+{: .bs-callout .bs-callout-info}
+The `mergeClasses()` function is used for merging a component's default classes with custom class overrides.
+
+**Change 3:** Set the `className` property of the `div` container to the `root` class:
+
+```diff
+-return(<div>{tagList}</div>)
++return(<div className={classes.root}>{tagList}</div>)
+```
+
+### Define the Tag styles
+
+Tags need to stand out from the surrounding text and look button-like.
+
+#### `tag.css`
+
+Open the `tag.css` file and add the following content:
+
+```css
+.root {
+    border: solid 1px #2680eb;
+    padding: 3px 9px;
+    margin: 5px;
+    border-radius: 5px;
+}
+
+.content {
+    color: #2680eb;
+    font-size: 0.875rem;
+}
+```
+
+#### `tag.js`
+
+Open the `tag.js` file and make the following changes to the component.
+
+**Change 1:** Import the `mergeClasses()` function and the `tag.css` file:
+
+```diff
+ import Button from '../Button';
++import { mergeClasses } from '../../classify';
++import defaultClasses from './tag.css';
+ import { Link } from '@magento/venia-drivers';
+```
+
+**Change 2:** Use the `mergeClasses()` function to merge custom classes passed into the component and create a new `buttonClasses` object:
+
+```diff
+ const { value } = props;
+ 
++const classes = mergeClasses(defaultClasses, props.classes);
++
++const buttonClasses = {
++    root_lowPriority: classes.root,
++    content: classes.content
++};
+```
+
+The `buttonClasses` object is used to override the styles in the Button component.
+The component expects a specific shape for the `classes` prop that the `buttonClasses` object fits.
+
+**Change 3:** Set the styles for the Link and Button component:
+
+```diff
+ return (
+-    <Link to={url}>
+-        <Button priority="low" type="button">
+-            {name}
+-        </Button>
+-    </Link>
++   <Link className={classes.link} to={url}>
++       <Button classes={buttonClasses} priority="low" type="button">
++           {name}
++       </Button>
++   </Link>
+ );
+```
+
+## Congratulations
+
+You just created a new storefront components using PWA Studio libraries and conventions.
```

Comment:
🎉 

May want to show that the click functionality / `url_path` works by including a screenshot after click?